pkg/corerp/frontend/controller/secretstores/kubernetes.go

```diff
@@ -317,3 +332,11 @@ func getSecretFromOutputResources(resources []rpv1.OutputResource, options *cont
 
 	return ksecret, nil
 }
+
+func isGlobalScopedResource(resource *datamodel.SecretStore) bool {
+	if resource.Properties.Application == "" && resource.Properties.Environment == "" {
+		return true
+	}
+
+	return false
```

Comment:
```suggestion
	return resource.Properties.Application == "" && resource.Properties.Environment == ""
```

Reply:
I would move this function to BaseResourceProperties like EqualLinkedResource because this is the function for application and environment in BaseResourceProperties and can be referenced by the other codes:

https://github.com/radius-project/radius/blob/c52aa0034c335fd2374ae4fe8214d937de0197fd/pkg/rp/v1/types.go#L54-L56

Reply:
added it